resources/megameklab/resources/Views.properties

```diff
@@ -245,22 +245,22 @@ MovementView.cbJumpType.text=Jump Type:
 MovementView.cbJumpType.tooltip=Non-ground movement equipment.
 MovementView.cbJumpType.null=None
 
-FuelView.spnFuel.text=Fuel Mass:
+FuelView.spnFuel.text=Fuel Mass: 
 FuelView.spnFuel.tooltip=The amount of mass allocated to fuel storage. For small support vehicles this is in kg.
-FuelView.lblFuelPoints.text=Fuel Points:
+FuelView.lblFuelPoints.text=Fuel Points: 
 FuelView.lblFuelPoints.tooltip=The number of fuel points provided by the allocated fuel.
-FuelView.lblRange.text=Range (km)
+FuelView.lblRange.text=Range (km): 
 FuelView.lblRange.tooltip=The vehicle's operating range for the given fuel capacity, in kilometers
-FuelView.lblTurnsAtSafe.text=Turns at Safe:
+FuelView.lblTurnsAtSafe.text=Game Turns at Safe Thrust: 
 FuelView.lblTurnsAtSafe.tooltip=The number of game turns the aircraft can operate at safe thrust before expending fuel.
-FuelView.lblTurnsAtMax.text=Turns at Max:
+FuelView.lblTurnsAtMax.text=... at Max: 
 FuelView.lblTurnsAtMax.tooltip=The number of game turns the aircraft can operate at maximum thrust before expending fuel.
-FuelView.lblBurnDays1G.text=Burn Days 1G:
+FuelView.lblBurnDays1G.text=Burn Days 1G: 
 FuelView.lblBurnDaysStationKeeping.text=Burn Days (Station-keeping):
 FuelView.lblBurnDays1G.tooltip=The number of days the craft can operate at 1G acceleration, or 0.1G for units with station-keeping drives.
-FuelView.lblBurnDaysMax.text=Burn Days Max:
+FuelView.lblBurnDaysMax.text=Burn Days Max: 
 FuelView.lblBurnDaysMax.tooltip=The number of days the craft can operate at maximum acceleration.
-FuelView.cbFuelType.text=Fuel Type:
+FuelView.cbFuelType.text=Fuel Type: 
```

Comment:
You'll want to use \u0020 instead of a trailing space, cause I've seen the trailing space automatically removed by the IDE.

Reply:
These resource files are the one place where I wish I could use "  " ...

Reply:
I concur completely.